server/log_rpc_server_test.go

```diff
@@ -1377,29 +1377,12 @@ func TestGetConsistencyProofBeginTXFails(t *testing.T) {
 	test.executeBeginFailsTest(t)
 }
 
-func TestGetConsistencyProofGetTreeRevision1Fails(t *testing.T) {
-	ctrl := gomock.NewController(t)
-	defer ctrl.Finish()
-
-	test := newParameterizedTest(ctrl, "GetConsistencyProof", readOnly,
-		func(t *storage.MockLogTX) {
-			t.EXPECT().GetTreeRevisionIncludingSize(getConsistencyProofRequest25.FirstTreeSize).Return(int64(0), int64(0), errors.New("STORAGE"))
-		},
-		func(s *TrillianLogRPCServer) error {
-			_, err := s.GetConsistencyProof(context.Background(), &getConsistencyProofRequest25)
-			return err
-		})
-
-	test.executeStorageFailureTest(t)
-}
-
 func TestGetConsistencyProofGetTreeRevision2Fails(t *testing.T) {
```

Comment:
Renamed. It's when you fail to look up the revision for snapshot2 in getting a consistency proof.